gitgud/levels/reworking/_interactiverebase/instructions.txt

```diff
@@ -0,0 +1,13 @@
+This practice level will familiarize with using interactive rebasing.
+>>>
+This is an alternative way of using the "rebase" command with "-i" right after it. For example, "git rebase -i HEAD~4" allows you to change the last 4 commits with the text editor.
+>>>
+When you include this option, git will open a text editor to show which commits will be copied below the rebase target.
+In the text editor, you're given the ability to do three things: reorder commits, omit commits, and squash commits(combine commits).
+>>>
+In order to reorder commits, simply change the order of the commits in the text editor.
+In order to omit a commit, simply delete the "pick" before the commit.
```

Comment:
I would tell them to use the word "drop" or the letter "d", but you can also mention that deleting the line has the same effect. Deleting the command only does not work

Reply:
Clarification:
```
pick a2a7df9
```
is the same as
```
a2a7df9
```

And we should say that. 

Omitting is the same as dropping or deleting the entire line